bundles/org.openhab.binding.roku/src/main/resources/OH-INF/thing/roku.xml

```diff
@@ -209,4 +229,10 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="powerState">
+		<item-type>String</item-type>
+		<label>Power State</label>
+		<description>Power State of the TV</description>
```

Comment:
```suggestion
		<description>Power state of the TV</description>
```

See also comment in README regarding this channel type. If this channel is really needed, perhaps some additional description would be helpful? Probably it should also be marked as advanced, since I would assume most users would be satisfied with the bi-directional `power` channel.